packages/core/src/main/notification/NotificationService.ts

```diff
@@ -0,0 +1,59 @@
+import {CRUDEngine} from '@wireapp/store-engine/dist/commonjs/engine/index';
+import APIClient = require('@wireapp/api-client');
+import {Notification} from '@wireapp/api-client/dist/commonjs/notification/index';
+import {RecordNotFoundError} from '@wireapp/store-engine/dist/commonjs/engine/error';
+
+export default class NotificationService {
+  public static STORES = {
+    AMPLIFY: 'amplify',
```

Comment:
Looking back it was a terrible decission to name the table `amplify`. Do we need that name for backward compatibility or do we have a chance now to change it?